### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by rafaelfaustini**
# Description

In this pullrequest I want to help the code legibility and do some small changes that may result in better performace and in order to avoid logical errors.

# Tests

Since no big changes were made, the tests seem that the plugin is working without problems

 ## Tested with dependencies : 
 - LSPDFR 0.4.4
- Arrest Manager 7.11.0.0
- Traffic Policer  6.16.0.0

# Checklist:

- [ X ] Ensure that classes attributes will not be changed directly
- [ ] Improve the code legibility

### CODE REVIEW

File: LSPDFR+/LSPDFR+.csproj [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>LSPDFR_</RootNamespace>
     <AssemblyName>LSPDFR+</AssemblyName>
-    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.7.1</TargetFrameworkVersion>
```
Comment (Albo1125) [inline LSPDFR+/LSPDFR+.csproj]: I believe RPH targets 4.6.1. Probably best to leave this as is.

File: LSPDFR+/LSPDFR+.csproj [hunk 2]
```diff
@@ -69,25 +69,27 @@
       <HintPath>dependencies\Albo1125.Common.dll</HintPath>
       <SpecificVersion>False</SpecificVersion>
     </Reference>
-    <Reference Include="Arrest Manager, Version=7.9.1.0, Culture=neutral, processorArchitecture=AMD64">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>dependencies\Arrest Manager.dll</HintPath>
+    <Reference Include="Arrest Manager">
```
Comment (Albo1125) [inline LSPDFR+/LSPDFR+.csproj]: Please don't commit changes to the dependencies with path changes.

**Comment by Albo1125** [comment]
Thanks for the refactoring work :)

File: LSPDFR+/CourtSystem.cs [hunk 3]
```diff
@@ -358,18 +281,23 @@ public static DateTime DetermineCourtHearingDate()
...
+        public int GuiltyChance { get; set; }
+        public string CourtVerdict { get; set; }
+        public DateTime ResultsPublishTime { get; set; }
+        public bool ResultsPublished { get; set; }
+        public bool ResultsPublishedNotificationShown { get; set; } = false;
+
+        public static bool PendingResultsMenuCleared { get; set; } = false;
+        public static bool ResultsMenuCleared { get; set; } = false;
+
+        public string getSuspectName()
```
Comment (Albo1125) [inline LSPDFR+/CourtSystem.cs]: What is the point of this given SuspectName is a getset property?

File: LSPDFR+/Main.cs [hunk 4]
```diff
@@ -35,20 +35,20 @@ public override void Initialize()
...
-        internal static float MinimumRPHVersion = 0.51f;
+        internal const float MinimumRPHVersion = 0.51f;
         internal static string[] AudioFilesToCheckFor = new string[] { };
         internal static string[] OtherFilesToCheckFor = new string[] {  }; //"Plugins/LSPDFR/LSPDFR+/CourtCases.xml"
         internal static Version RAGENativeUIVersion = new Version("1.6.3.0");
-        internal static Version MadeForLSPDFRVersion = new Version("0.4.2");
+        internal static Version MadeForLSPDFRVersion = new Version("0.4.4");
 
-        internal static string DownloadURL = "https://www.lcpdfr.com/files/file/11930-lspdfr-improved-pursuit-ai-better-traffic-stops-court-system/";
+        internal const string DownloadURL = "https://www.lcpdfr.com/files/file/11930-lspdfr-improved-pursuit-ai-better-traffic-stops-court-system/";
```
Comment (Albo1125) [inline LSPDFR+/Main.cs:46]: How did I not think of this before

**Review by Albo1125 — CHANGES_REQUESTED** [review]

**Review by Albo1125 — APPROVED** [review]

File: LSPDFR+/CourtSystem.cs (hunk 3 above)
Comment (rafaelfaustini) [reply LSPDFR+/CourtSystem.cs]: This getSuspectName is not suposed to exist, just the { get; set;}, my mistake

**Pull request merged**